src/pages/dashboard/index.js

```diff
@@ -11,93 +11,90 @@ import { getPosts, getUserByName, getUsers } from "../../service/apiClient"
 import UsersList from "../../components/usersList"
 
 const Dashboard = () => {
-    const [searchVal, setSearchVal] = useState("")
-    const [posts, setPosts] = useState([])
-    const [users, setUsers] = useState([])
+  const [searchVal, setSearchVal] = useState("")
+  const [posts, setPosts] = useState([])
+  const [users, setUsers] = useState([])
 
-    const sortPosts = (fetchedPosts) => {
-        const sortedPosts = fetchedPosts.sort(
-            (a, b) => new Date(b.createdAt) - new Date(a.createdAt)
-        )
-        return sortedPosts
-    }
+  const sortPosts = (fetchedPosts) => {
+    const sortedPosts = fetchedPosts.sort(
+      (a, b) => new Date(b.createdAt) - new Date(a.createdAt)
+    )
+    return sortedPosts
+  }
 
-    const getAllPosts = () => {
-        getPosts()
-            .then(sortPosts)
-            .then(setPosts)
-            .catch((error) => {
-                console.error("Fetch error:", error.message)
-            })
-    }
+  const getAllPosts = () => {
+    getPosts()
+      .then(sortPosts)
+      .then(setPosts)
+      .catch((error) => {
+        console.error("Fetch error:", error.message)
```

Comment:
let's give this more context. Rather than "fetch error", explicitly say "error getting all posts"